creme/reports/tests/fake_models.py

```diff
@@ -37,13 +38,33 @@ def __str__(self):
         def get_absolute_url(self):
             return reverse('reports__view_fake_folder', args=(self.id,))
 
+    class FakeReportsColorTag(CremeEntity):
+        title = models.CharField(_('Title'), max_length=100)
+
+        color = core_fields.ColorField(
+            default=core_fields.ColorField.random,
+        )
+
+        class Meta:
+            app_label = 'reports'
+            verbose_name = 'Test (reports) Color Tag'
+            verbose_name_plural = 'Test (reports) Color Tags'
+            ordering = ('title',)
+
+        def __str__(self):
+            return self.title
+
```

Comment:
- I presume it could be a simple CremeModel (or better, a Minion)
- I prefer a name like "Category" (vanilla Document gets a field "categories"); the term "Tag" has already another meaning in Creme.
Do not forget to modify the migration file too.